test/include/dlaf_test/util_matrix.h

```diff
@@ -64,9 +64,31 @@ std::vector<hpx::future<Tile<T, device>>> getFutures(Matrix<T, device>& mat) {
   return result;
 }
 
+/// @brief Returns a col-major ordered vector with the futures to the matrix tiles.
+template <class T, Device device>
+std::vector<hpx::future<Tile<T, device>>> getFuturesUsingGlobal(Matrix<T, device>& mat) {
+  const matrix::Distribution& dist = mat.distribution();
+
+  std::vector<hpx::future<Tile<T, device>>> result;
+  result.reserve(util::size_t::mul(dist.localNrTiles().rows(), dist.localNrTiles().cols()));
+
+  for (SizeType j = 0; j < dist.nrTiles().cols(); ++j) {
+    for (SizeType i = 0; i < dist.nrTiles().rows(); ++i) {
+      comm::Index2D owner{dist.rankGlobalTile<Coord::Row>(i), dist.rankGlobalTile<Coord::Col>(j)};
```

Comment:
Why you don't use `rankGlobalTile(GlobalTileIndex)`?

Otherwise you can keep the calculation for row and column and
and skip the whole i loop if the j column is not in this rank.